### DISCUSSION TITLE
Is there a way to specify the --wait option?

### DISCUSSION THREAD

**Discussion by stephanebastos** [helmfile/helmfile#809] [Questions]
Hello everyone,
I tried to install a job with the --wait-for-jobs and it work as expected if the job complete wihtout a problem.
But if during the installation, for any reason , the job crashes, helmfile detects that the job failed but wait for the entire duration of the timeout.
Is it possible to set a specific condition? or any condition really to avoid an unecessary amount of time? (it's valable for any ressource type, not only job type)

Thanks in advance!

**Comment by yxxhero** [comment]
maybe this is a feature request of the helm.
